src/modules/pmc/tests/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+project(pmctests)
+
+cmake_minimum_required(VERSION 3.2.0)
+
```

Comment:
This is now specified in the root CMakeList.txt for the project. Let's omit it from here.

Reply:
All other tests files have this line, that's why I left it. But now I've deleted it.